apps/docs/content/docs/components/range-calendar.mdx

```diff
@@ -236,7 +236,7 @@ Here's the example to customize `topContent` and `bottomContent` to have some pr
 | Attribute              | Type                            | Description                                                                                                                              |
 | ---------------------- | ------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
 | onFocusChange          | `(date: CalendarDate) => void`  | Handler that is called when the focused date changes.                                                                                    |
-| onChange               | `(value: RangeValue>) => void`  | Handler that is called when the value changes.                                                                                           |
+| onChange               | `(value: RangeValue<MappedDateValue<DateValue>>) => void`  | Handler that is called when the value changes.                                                                                           |
```

Comment:
@wingkwong we should add the `RangeValue` and `MappedDateValue` as a reference above the supported calendars